nessus-agent/elx/init.sls

```diff
@@ -30,6 +30,75 @@ Print nessus-agent help:
         without linking the agent to the server.
 {%- endif %}
 
+{%- if (nessus.log_rotation_strategy) %}
+Add Log Rotation Strategy:
+  file.replace:
+    - name: '/opt/nessus_agent/var/nessus/log.json'
+    - pattern: '"file": "/opt/nessus_agent/var/nessus/logs/www_server.log"'
+    - repl: '"rotation_strategy": {{nessus.log_rotation_strategy}},\n\t\t\g<0>'
```

Comment:
You can get the double-quotes around the value by adding them here, instead of requiring them get passed through the pillar... e.g. `"{{nessus.log_rotation_strategy}}"`

Reply:
I removed it altogther since it looks like `file.replace` automatically inserts the double quotes into the `log.json` file.

Reply:
ehh, `file.replace` really shouldn't automatically be doing anything of the sort. i wrote a good chunk of that function... what was the pillar set to?

Reply:
I'm just guessing what `file.replace` is doing and have no idea what adds the double quotes.  😁 

With my latest changes, the pillar is set as follows:
```
RedHat:
  nessus-agent:
    lookup:
      log_rotation_strategy: 'daily'
```
The double quotes is not needed in both the pillar and the `file.replace` definition.

Originally, I had the pillar as follows:
```
RedHat:
  nessus-agent:
    lookup:
      log_rotation_strategy: '"daily"'
```
This adds the text into the `log.json` as expected.

However, I did try what you suggested to add the double quotes in `repl` but what ended up happening is there would be two sets of double quotes in the json file.

Reply:
Disregard my comments above.  Wound up being user error...